block/manager.go

```diff
@@ -85,6 +86,16 @@ type Manager struct {
 
 	// indexer
 	indexerService *txindex.IndexerService
+
+	syncingC chan struct{}
+
+	validateC chan struct{}
+
+	synced *uchannel.Nudger
```

Comment:
i think there is a confusion created because i forgot to rename the function. syncing it just happens from the syncloop. syncFromSettlement is only retrieving latest batch info when starting. i renamed the function and added doc to make it clear